modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcasts/FolderAdapter.kt

```diff
@@ -178,12 +179,15 @@ class FolderAdapter(
 
         val button: View = view.findViewById(R.id.button)
         val podcastThumbnail: ImageView = view.findViewById(R.id.podcast_artwork)
+        val podcastCardView = view.findViewById<CardView>(R.id.podcast_card_view)
         val podcastBackground: View? = view.findViewById(R.id.header_background)
         val podcastTitle: TextView = view.findViewById(R.id.library_podcast_title)
         val author: TextView? = view.findViewById(R.id.podcast_author)
         val unplayedText: TextView = view.findViewById(R.id.unplayed_count)
         val unplayedBackground: ImageView? = view.findViewById(R.id.unplayed_background)
         val countTextMarginSmall: Int = 2.dpToPx(view.resources.displayMetrics)
+        val cardElevation: Float = 2.dpToPx(view.resources.displayMetrics).toFloat()
+        val cardCornerRadius: Float = 4.dpToPx(view.resources.displayMetrics).toFloat()
```

Comment:
Really small suggestion: What do you think we add a top padding for the badge? 

I feel the new drop-shadow is clipping a bit the podcast above. See:

![Screenshot_20240508_093732_Pocket Debug~2](https://github.com/Automattic/pocket-casts-android/assets/42220351/c404236b-bc3b-4fb5-aa33-fb3f5d739f45)

and see the proposal [here](https://github.com/Automattic/pocket-casts-ios/issues/1690)